### PR TITLE
chore(types): add tests for the types

### PR DESCRIPTION
**Pull request by boneskull**
- **feat(types): adds some types**
- **chore(types): add tests for the types**

### CODE REVIEW

**Comment by boneskull** [comment]
Oh, well, I meant to target #16.  But mostly I just wanted to show that the types are correct.

**Comment by boneskull** [comment]
Here's [a workflow that passed](https://github.com/boneskull/loggerr/actions/runs/11655632490)

**Comment by wesleytodd** [comment]
I will just close the other PR and we can use this. I don't mind merging this as two commits as is.

File: loggerr.d.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
+
+declare module 'loggerr' {
+  export type Formatter = 'browser' | 'bunyan' | 'cli' | 'default';
```
Comment (wesleytodd) [inline loggerr.d.ts:3]: I recently added a new `cli-simple` https://github.com/wesleytodd/loggerr/blob/main/formatters/cli-simple.js

File: test/types.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+expectTypeOf<Loggerr['DEBUG']>().toEqualTypeOf<6>();
+
+const defaultLogger = new Loggerr();
+
+expectTypeOf<(typeof defaultLogger)['level']>().toBeNumber();
+
+expectTypeOf<(typeof defaultLogger)['debug']>().toMatchTypeOf<LogFunction>();
+
+const customLogger = new Loggerr({ levels: ['butts', 'feet', 'armpits'] });
```
Comment (wesleytodd) [inline test/types.ts:24]: 🤣

**Review by wesleytodd — APPROVED** [review]

File: loggerr.d.ts (hunk 1 above)
Comment (wesleytodd) [inline loggerr.d.ts:3]:
```suggestion
  export type Formatter = 'browser' | 'bunyan' | 'cli' | 'cli-simple' | 'default';
```

**Comment by boneskull** [comment]
`cli-simple` was not documented so I was unsure if it was public.

**Comment by boneskull** [comment]
@wesleytodd Also, the exports are wrong, I'll send another PR to fix it.

**Pull request merged**